aspnetcore-webapi/SerializationDemo/SerializationDemo.EmployeeApi/Controllers/EmployeesController.cs

```diff
@@ -0,0 +1,33 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using SerializationDemo.Common.Models;
+using SerializationDemo.EmployeeApi.Repositories;
+
+namespace SerializationDemo.EmployeeApi.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class EmployeesController : ControllerBase
+    {
+        private readonly IEmployeeRepository _repo;
+
+        public EmployeesController(IEmployeeRepository repo)
+        {
+            _repo = repo;
+        }
+
+        [HttpGet("")]
+        [ProducesResponseType(typeof(IEnumerable<Employee>), StatusCodes.Status200OK)]
+        public IActionResult GetAllEmployees()
+        {
+            return Ok(_repo.GetAll());
+        }
+
+        [HttpPost("")]
```

Comment:
You don't need ("") after HttpGet and HttpPost if you don't want to change the default route. [HttpGet] works just fine.

Reply:
done